worker/pkg/query-builder/insert-query-builder.go

```diff
@@ -151,7 +158,8 @@ func (d *PostgresDriver) buildInsertQuery(
 	rows []map[string]any,
 ) (sql string, args []any, err error) {
 	goquRows := toGoquRecords(rows)
-	if d.options.conflictConfig.onConflictDoUpdate != nil {
+	// if the table has a deferrable constraint, we can't use on conflict do update
```

Comment:
from the testing I did earlier on conflict do nothing can trigger the issue too, not just do update.

Reply:
true but only when using default transformer. passthrough works just fine